docs/extensions/guides/renderer-extension.md

```diff
@@ -317,6 +306,87 @@ The `title` and `components.Description` fields appear on the cluster settings p
   abstract updateStatus(cluster: Cluster): Promise<ClusterFeatureStatus>;
 ```
 
+The `install()` method is typically called by Lens when a user has indicated that this feature is to be installed (i.e. clicked **Install** for the feature on the cluster settings page). The implementation of this method should install kubernetes resources using the `applyResources()` method, or by directly accessing the kubernetes api ([`K8sApi`](tbd)).
```

Comment:
I would leave out the word "typically"

Reply:
I want to convey that it is not necessarily the only way `install()` gets called (Lens could now or in the future invoke it in another context. Could be invoked by other extension code...) Maybe there's a better word or phrase (can't be very long though)